core/src/test/java/org/apache/camel/kafkaconnector/CamelSourceTaskTest.java

```diff
@@ -179,7 +182,7 @@ public void testSourcePollingTimeout() throws InterruptedException {
         CamelSourceTask camelSourceTask = new CamelSourceTask();
         camelSourceTask.start(props);
 
-        long sleepTime = 30L;
+        long sleepTime = 11L;
```

Comment:
Is there a specific reason?

Reply:
While running tests, I found `30L` to fail sporadically.

Reply:
If that is the case I think it should be addressed in a separate PR after opening an issue.

Reply:
Reverted